### PR TITLE
Typo && Spacing

### PR DESCRIPTION
**Pull request by BusbyActual**
There was an extraneous space. I also updated functions syntax to follow the style guide referenced in other parts of this document. These functions could potentially be updated to es6 as well.

### CODE REVIEW

File: sections/errorhandling/failfast.md [hunk 1]
```diff
@@ -29,11 +28,11 @@ function addNewMember(newMember)
 ### Anti-pattern: no validation yields nasty bugs
 
 ```javascript
-// if the discount is positive let's then redirect the user to pring his discount coupons
-function redirectToPrintDiscount(httpResponse, member, discount)
-{
-    if(discount != 0)
+// if the discount is positive let's then redirect the user to print his discount coupons
+function redirectToPrintDiscount(httpResponse, member, discount)
```
Comment (sven-piller) [inline sections/errorhandling/failfast.md]: Is the opening bracket of this function missing or is the diff view misleading?

Comment (BrunoScheufler) [reply sections/errorhandling/failfast.md]: It's actually missing, just looked up the file in its current state

**Comment by i0natan** [comment]
@BrunoScheufler Going by our workflow, shouldn't we assign an assignee?

**Comment by BrunoScheufler** [comment]
@i0natan we'll have to open a chores issue for December, since the last one is for November, and we've got new flowers to assign 😄

**Comment by BusbyActual** [comment]
@BrunoScheufler happy to help with that just ping me 👍

**Pull request merged**